### ISSUE TITLE
github repo clean up

### ISSUE THREAD

**Issue by harshcrop** [beckn/missions#101]
I've completed the process of changing 12 repositories from public to private. I've also maintained a list of the repositories that were updated to private status. I'm currently reviewing additional repositories to identify any that are empty.

Report sheet - https://docs.google.com/spreadsheets/d/1sTKyop1a662JchZk4JDcQVJWZ7x1xMoEXoqbRHt8_fQ/edit?gid=0#gid=0

**Comment by harshcrop** [comment]
On Aug 19-30 iteration i have completed the below items. Since the iteration is ended up and there are some tasks to taken care. I am closing this card and in the current iteration.

Completed Details

I've completed the process of changing 12 repositories from public to private. I've also maintained a list of the repositories that were updated to private status. I'm currently reviewing additional repositories to identify any that are empty.

Report sheet - https://docs.google.com/spreadsheets/d/1sTKyop1a662JchZk4JDcQVJWZ7x1xMoEXoqbRHt8_fQ/edit?gid=0#gid=0

Pending Details []

- [working on more repo audit for cleanup]
